Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.yml

```diff
@@ -100,6 +100,12 @@ configuration:
   type: 8
   additionalinfo: 'A parameter integration that removes the playbook ID field from incoming incidents. Note: When set to true (default), the instance will attempt to run a playbook according to the incoming ID. When set to false, the instance will run the default playbook for the incident type (if configured locally).'
   required: false
+- display: Reset mirror 
+  name: reset_mirror
```

Comment:
This name is no good.
It should be something like "Fetch incident history" - since this is the effect of this flag - to pull the entire history, rather than just the delta.

Reply:
perhaps Mirror incident history ?